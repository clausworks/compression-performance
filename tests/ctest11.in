files/ctest01 -n2
