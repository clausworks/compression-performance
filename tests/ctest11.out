Exited with code 64
